.github/workflows/main.yml

```diff
@@ -61,58 +61,98 @@ jobs:
           docker build --file ./microservices/embeddingservice/docker/Dockerfile ./microservices/embeddingservice
           echo "embedding service image build OK"
 
-  
-  docker_build_publish_dev:
+
+  build_publish_deploy_ingest_dev:
     needs: unit_tests
```

Comment:
Depending on reco explorer unit test does not make sense to me.
Wouldn't we want to run microservice unit tests here?

Reply:
good catch